foss4gna_24/big-EO-viz.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>Minor - Yuvi built the Notebook Sharing not Sanjay</p><p> </p><p>Maybe "for more information on the Lidar processing see the geotrees-demo repo" ...</p><p> </p><p>Do we need to mention that this only works on MAAP, and put MAAP in the Additional resources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Noted with thanks. updated the notebook</p>